scenario.md

```diff
@@ -1,17 +1,21 @@
 # Scenario Questions #
-*These questions have no right and wrong answer. They should allow the candidate to provide full responses. The interviewers here are not just looking for complete answers but also keeping track of HOW a candidate responds (rambles, gets distracted, doesn't answer the question, etc.).*
+
+o right and wrong answer. They should allow the candidate to provide full responses. The interviewers here are not just looking for complete answers but also keeping track of HOW a candidate responds (rambles, gets distracted, doesn't answer the question, etc.).*

```

Comment:
Looks like you may have removed the beginning of this paragraph. Can you fix (add the old content back) and resubmit?